pages/Documents/ConversationalAI/Third-PartyBots/getting-started.md

```diff
@@ -39,6 +39,9 @@ Each connector provides the ability to:
 {: .important}
 Some connectors may provide more or less functionality depending on the specifics of the provider.
 
+{: .important}
+Please be advised that you should create fresh bot agents for your bots. Using the same bot agents for ConversationBuilder and ThirdPartyBots will break both bot instances. This happens because only one active user session at time is possible within the ConversationalCloud ecosystem, so bots using the same bot agent will kick each other out.
```

Comment:
Please be advised that you should create fresh bot agents for your bots. Using the same bot agents for ConversationBuilder and ThirdPartyBots will break both bot instances. Conversational Cloud only allows one active user session per agent. Thus, bots created in Conversation Builder and Third-Party Bots with the same bot agent will eventually kick each other out.